source/ceylon/test/engine/internal/file.ceylon

```diff
@@ -0,0 +1,39 @@
+import java.io {
+    File,
+    FileOutputStream,
+    PrintWriter
+}
+
+native class FileFacade(String path) satisfies Destroyable {
```

Comment:
Or, put differently: Would you mind if I change the facade to a class, and moved it to `file.ceylon`? I think the first change would make it cleaner, and the second change would avoid that someone else makes the same mistake as me in the future :)